packages/calling/src/common/Utils.ts

```diff
@@ -1110,8 +1111,13 @@ export async function getXsiActionEndpoint(
 
         let xsiEndpoint = response[DEVICES][0][SETTINGS][BW_XSI_URL];
 
-        if (response[DEVICES][0][SETTINGS][BW_XSI_URL].endsWith(BW_XSI_ENDPOINT_VERSION)) {
-          xsiEndpoint = response[DEVICES][0][SETTINGS][BW_XSI_URL].slice(0, -5);
+        const xsiUrl = response[DEVICES][0][SETTINGS][BW_XSI_URL];
+
+        // Check if it ends with specific version and slice accordingly
+        if (xsiUrl.endsWith(BW_XSI_ENDPOINT_VERSION)) {
+          xsiEndpoint = xsiUrl.slice(0, -5); // Remove 'v2.0'
```

Comment:
Rather than making it specific to version 2, can we use regex to find if it ends with a version, otherwise we would have to re-visit this once the version changes to v3

Reply:
<img width="1234" alt="Screenshot 2024-10-14 at 10 44 32 AM" src="https://github.com/user-attachments/assets/517fdb97-a863-4391-9e4b-97c055f32128">

Hey Shreyas, as per the 
Broadworks solution guide version will be v2
https://help.webex.com/en-us/article/no4dlj3/Webex-for-Cisco-BroadWorks-Solution-Guide#reference-template_9e47637b-bf3e-4cd4-b19e-89fbc4e0e7d1

This will not change, but we will reach out to Broadworks PM for confirmation and make the necessary changes in the next PR.